logic/src/commonTest/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepositoryTest.kt

```diff
@@ -819,8 +825,9 @@ class MLSConversationRepositoryTest {
                 .then { NetworkResponse.Success(SendMLSMessageResponse(TIME, events), emptyMap(), 201) }
         }
 
-        fun withCommitBundleSuccessful() = apply {
-            withSendMLSMessageSuccessful()
+        fun withCommitBundleSuccessful(events: List<EventContentDTO> = emptyList()) = apply {
+            withSendMLSMessageSuccessful(events)
```

Comment:
This should also be removed I guess but then again I guess we can remove this whole function.